docs/troubleshoot/app-framework/app-known-issues.md

```diff
@@ -4,6 +4,29 @@ The following topics contain information that can help you troubleshoot problems
 
 Most of the solutions below identify issues by referring to the Zowe [logs](app-mustgather.md). To identify and resolve issues, you should become familiar with their names and locations.
 
+## Desktop apps fail to load
+
+**Symptom:**
+
+When you open apps in the desktop they display a page with the message "The plugin failed to load."
+
+**Solution:**
+
+NodeJS v8.16.1 performs auto-encoding in a way that breaks Zowe apps. See [https://github.com/ibmruntimes/node/issues/142](https://github.com/ibmruntimes/node/issues/142) for details.
+
+Use node v8.16.0 which is available at [https://www.ibm.com/ca-en/marketplace/sdk-nodejs-compiler-zos](https://www.ibm.com/ca-en/marketplace/sdk-nodejs-compiler-zos). Download the `ibm-trial-node-v8.16.0-os390-s390x.pax.Z` file.
+
+## NODEJSAPP disables immediately
+
+**Symptom:**
+
+If you receive the message CEE5207E The signal SIGABRT was received in stderr, you might have reached the limit for shared message queues on your LPAR.

```

Comment:
Only one comment on a minor issue. The message text should be highlighted. For example: 
If you receive the message `CEE5207E The signal SIGABRT was received` in stderr,......

Reply:
Thanks Ashley, I highlighted the message and pushed a new commit.

Reply:
FYI @1000TurquoisePogs and @jackjia-ibm it sounds like we're releasing 1.6.x today and publishing the doc so I'm going to merge this PR this afternoon. If you get time in the future to review and have changes just open a new issue and I'll update.